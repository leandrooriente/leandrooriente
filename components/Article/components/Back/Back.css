.link {
  display: inline-block;
  color: var(--secondary-color);
  border-bottom: 1px solid transparent;
  font-size: 18px;
  text-decoration: none;
  margin: 30px 0;
}

.link:hover {
  color: var(--highlight-color);
  border-color: var(--highlight-color);
  transition: all var(--fast-transition) ease-in;  
}