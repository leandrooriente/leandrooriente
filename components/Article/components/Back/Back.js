import React from 'react';
import Link from 'next/link';
import css from './Back.css';

const Back = () => (
  <Link href="/">
    <a className={css.link}>â†¤ Articles</a>
  </Link>
);

export default Back;
