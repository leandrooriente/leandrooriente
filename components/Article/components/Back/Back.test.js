import React from 'react';
import { mount } from 'enzyme';
import Back from './Back';

describe('<Back />', () => {
  it('should render <Back /> component', () => {
    const wrapper = mount(<Back />);

    expect(wrapper).toMatchSnapshot();
  });
});
