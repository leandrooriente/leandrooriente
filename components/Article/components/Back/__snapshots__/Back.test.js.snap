// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Back /> should render <Back /> component 1`] = `
<Back>
  <Link
    href="/"
  >
    <a
      className="link"
      href="/"
      onClick={[Function]}
    >
      â†¤ Articles
    </a>
  </Link>
</Back>
`;
